test/commands/ParsedCommand.spec.js

```diff
@@ -0,0 +1,60 @@
+import _isEqual from 'lodash/isEqual';
+import ava from 'ava';
+import CommandParser from '../../src/assets/scripts/client/commands/parsers/CommandParser';
+
+
```

Comment:
Please use a single line for spacing. Here and below too

Reply:
Done :)